appleseed-Installer/LicenseAgreeDlg.wxs

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--//**********************************************************************/
+// Copyright (c) 2021 appleseed.
+// All rights reserved.
+// 
+//**************************************************************************/
+// DESCRIPTION: Contains the Dll Entry stuff
+// AUTHOR: Omid Ghotbi (TAO) omid.ghotbi@gmail.com www.3dfine.com
+//***************************************************************************/-->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <UI Id="LicenseAgreeDialogUI">
+
+      <DialogRef Id="VersionSelectDlg"/>
+      
+      <Property Id="IAGREE" Secure="yes" />
+      <Property Id="License">Licensebtn</Property>
+      <Binary Id="Licensebtn" SourceFile="Resources/btAgree.png"/>
+      <Property Id="Back">backbtn</Property>
+      <Binary Id="backbtn" SourceFile="Resources/btBack.png"/>
+
+      <Dialog Id="LicenseAgreeDlg" Width="555" Height="416" Title="appleseed : License Agreement">
```

Comment:
It seems that for open source, MIT-licensed software, we should skip the any license agreement step (which would technically be a EULA), as discussed in this post: https://opensource.stackexchange.com/questions/2865/does-open-source-software-need-an-eula-to-protect-the-developer

I'd suggest to drop this step entirely (as well as the `LICENSE.rtf` file).